packages/cli/src/lib/configureStorybook.js

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs-extra'
+
+import { getPaths } from '.'
+
+/**
+ * Configure Storybook for the given template by creating a custom preview config
+ */
+export default function configureStorybook({ force }, newStorybookPreview) {
+  const { storybookPreviewConfig } = getPaths().web
+
+  let finalNewStorybookPreview
+  /**
+   *  Check if storybookPreviewConfig already exists.
+   *  Merge both files if it does.
+   *  By removing import react and export decorator from new config
+   *  And put new config inside current config after last import
+   */
+  if (fs.existsSync(storybookPreviewConfig)) {
+    if (force) {
+      fs.unlinkSync(storybookPreviewConfig)
+      finalNewStorybookPreview = newStorybookPreview
+    } else {
+      const CurrentStorybookConfig = fs
+        .readFileSync(storybookPreviewConfig)
+        .toString()
+
+      const newDecoratorsName = newStorybookPreview.match(
+        /export const decorators = \[(.*?)\]/
+      )[1]
+
+      const currentDecoratorsName = CurrentStorybookConfig.match(
+        /export const decorators = \[(.*?)\]/
+      )[1]
+
+      const finalDecoratorsExport = `export const decorators = [${currentDecoratorsName}, ${newDecoratorsName}]`
+
+      const insideNewStorybookConfigWithoutReactAndDecoration =
+        newStorybookPreview
+          .replace(/import *. as React from 'react'/, '')
+          .replace(/export const decorators = .*/, '')
+
+      const CurrentStorybookConfigWithoutDecoration =
+        CurrentStorybookConfig.replace(/export const decorators = .*/, '')
+
+      const ReverseCurrentStorybookConfig =
+        CurrentStorybookConfigWithoutDecoration.split('\n').reverse()
+
+      const indexOfLastImport = ReverseCurrentStorybookConfig.findIndex(
+        (value) => /import/.test(value)
```

Comment:
```suggestion
        (value) => /^import /.test(value)
```

Make this a little more specific to better guard against false positives